backend/tournesol/serializers/rating.py

```diff
@@ -21,10 +21,13 @@ class ContributorRatingSerializer(ModelSerializer):
     n_comparisons = SerializerMethodField(
         help_text="Number of comparisons submitted by the current user about the current video",
     )
+    comparison_date = SerializerMethodField(
+        help_text="Date of comparison(oldest / newest) to order ratings"
+    )
```

Comment:
The `comparison_date` and the comment `Date of the comparison` is redundant.

We can safely delete it. Moreover, we should avoid mentions oldest / newest as the usage is the responsability of the API consumer. Maybe this field will be displayed, and not used for ordering, etc.

I'll fix it.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1170/commits/e55a9b88fed1e41a852567b871e2d3ee22da3084.